src/object/srv_obj_migrate.c

```diff
@@ -3962,6 +3962,7 @@ obj_tree_lookup_cont(daos_handle_t toh, uuid_t co_uuid, daos_handle_t *cont_toh)
 	memset(&uma, 0, sizeof(uma));
 	uma.uma_id = UMEM_CLASS_VMEM;
 	cont_root = tmp_iov.iov_buf;
+	D_ASSERT(daos_handle_is_valid(cont_root->tcr_root_hdl));
```

Comment:
Some confused, since it is already opened, why need to open again?

Reply:
it is two levels tree, 1 level is container tree, each node is the root of obj tree.

Reply:
I see what you mean, the problem is XS0 open handle cannot be used for other XS, so other XS need to open it again.